manifests/kubernetes/deployment.yaml-e

```diff
@@ -0,0 +1,143 @@
+  - nodes/proxy # access /pods
+  - nodes/stats # access /pods
+  verbs:
+  - 'get'
+  - 'watch'
+  - 'list'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kepler-clusterrole-binding
+  namespace: kepler
+roleRef:
+  kind: ClusterRole
+  name: kepler-clusterrole
+  apiGroup: "rbac.authorization.k8s.io"
+subjects:
+- kind: ServiceAccount
+  name: kepler-sa
+  namespace: kepler
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kepler-sa
+  namespace: kepler
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kepler-exporter
+  namespace: kepler
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: exporter
+      app.kubernetes.io/name: kepler-exporter
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: exporter
+        app.kubernetes.io/name: kepler-exporter
+    spec:
+      hostNetwork: true
+      serviceAccountName: kepler-sa
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: feature.node.kubernetes.io/cpu-cpuid.HYPERVISOR
+                operator: DoesNotExist
+                values:
+      containers:
+      - name: kepler-exporter
+        image: quay.io/sustainable_computing_io/kepler:latest
+        imagePullPolicy: Always
+        resources:
+          requests:
+            memory: "400Mi"
+            cpu: "100m"
+        securityContext:
+          privileged: true
+        command:
+        - /usr/bin/kepler
+        - -address
+        - 0.0.0.0:9102
+        - -enable-gpu=true
+        - -enable-cgroup-id=true
+        - -v=1
+        ports:
+        - containerPort: 9102
+          hostPort: 9102
+          name: http
+        livenessProbe:
+          failureThreshold: 5
+          httpGet:
+            path: /healthz
+            port: 9102
+            scheme: HTTP
+          initialDelaySeconds: 10
+          periodSeconds: 60
+          successThreshold: 1
+          timeoutSeconds: 10
+        volumeMounts:
+        - mountPath: /lib/modules
+          name: lib-modules
+        - mountPath: /sys
+          name: tracing
+        - mountPath: /proc
+          name: proc
+        env:
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        - name: CPU_ARCH_OVERRIDE
+          value:
+      volumes:
+      - name: lib-modules
+        hostPath:
+          path: /lib/modules
+          type: Directory
+      - name: tracing
+        hostPath:
+          path: /sys
+          type: Directory
+      - name: proc
+        hostPath:
+          path: /proc-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host-host
```

Comment:
this line looks odd

Reply:
@rootfs that's what I meant in "#Anything else" comment of the PR, I have mitigated it for k3d; for kind it still creates an never ending chain of "-host" every time you run it.